src/Games/NexusMods.Games.AdvancedInstaller.UI/Resources/Language.pl.resx

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="utf-8"?>
+<root>
+	<resheader name="resmimetype">
+		<value>text/microsoft-resx</value>
+	</resheader>
+	<resheader name="version">
+		<value>1.3</value>
+	</resheader>
+	<resheader name="reader">
+		<value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+	</resheader>
+	<resheader name="writer">
+		<value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+	</resheader>
+	<data name="Footer_CANCEL" xml:space="preserve">
+		<value>ANULUJ</value>
+	</data>
+	<data name="Footer_INSTALL" xml:space="preserve">
+		<value>ZAINSTALUJ</value>
+	</data>
+	<data name="OverlayHeaderText_Advanced_Installer" xml:space="preserve">
+		<value>Zaawansowany instalator</value>
+	</data>
+	<data name="InstallerNameText" xml:space="preserve">
+		<value>Zaawansowany ręczny instalator modyfikacji</value>
+	</data>
+	<data name="SubHeadingText_Please_select_files" xml:space="preserve">
+		<value>Wybierz pliki do zainstalowania i lokalizację, w której mają zostać zainstalowane.</value>
+	</data>
+	<data name="ModContentTitle_MOD_CONTENT" xml:space="preserve">
+		<value>ZAWARTOŚĆ MODYFIKACJI</value>
+	</data>
+	<data name="ModContentSubHeading_Files_with_no" xml:space="preserve">
+		<value>Pliki bez wybranej lokalizacji nie zostaną zainstalowane.</value>
+	</data>
+	<data name="CreateFolderButton_Create_new_folder" xml:space="preserve">
+		<value>Utwórz nowy folder</value>
+	</data>
+	<data name="NewFilePill_NEW" xml:space="preserve">
+		<value>NOWY</value>
+	</data>
+	<data name="Pill_DUPE_FOLDER" xml:space="preserve">
+		<value>DUPLIKUJ FOLDER</value>
```

Comment:
It's misleading in the original text, so apologies for that.

This is misleading in the original English file, so apologies for this/

This in the UI is used for a 'pill' (indicator) that is displayed if a folder is inside a folder of the same name.

This should say something like 'DUPLIKAT FOLDERU', because it's marking a duplicate in the UI to the user.

![20231111_01h26m04s_grim](https://github.com/Nexus-Mods/NexusMods.App/assets/6697380/4c3c57cf-8191-4112-ae77-90132f2c7bff)